web-client/src/presenter/computeds/TrialSession/addTrialSessionInformationHelper.ts

```diff
@@ -31,12 +32,18 @@ export const addTrialSessionInformationHelper = (
     });
   }
 
+  // NOTE: what happens if isStandaloneSession is true and user is docketclerk?
+  const getSessionTypes = (user: AuthUser) => {
+    return user.role === 'docketclerk'
+      ? ['Special', 'Motion/Hearing']
```

Comment:
Since the array `['Special', 'Motion/Hearing']` is used twice in this computed on lines 31 and 38, could it be extracted as a variable and given a descriptive name (as in the `docketClerkCanEditCheck` function included with this PR)?

Reply:
Yep, good call

Reply:
That's a great point! In that case, my instinct would be to create two separate variables, one in `getSessionTypes` and the other in the anonymous function passed to `filter` on line 30, each with a descriptive name. But that might be overkill.

Reply:
Ah, nice. I can do that! I do like adding code as docs